src/main/java/bio/terra/common/DaoKeyHolder.java

```diff
@@ -5,16 +5,17 @@
 import java.sql.Timestamp;
 import java.time.Instant;
 import java.util.Map;
+import java.util.Optional;
 import java.util.UUID;
 
 public class DaoKeyHolder extends GeneratedKeyHolder {
 
     public UUID getId() {
```

Comment:
I was thinking these methods would all be changed to return `Optional` e.g. `Optional<UUID>`. Although that would push the null check into the caller. Maybe something for a future PR.